src/github.com/eugenmayer/concourse-static-resource/in/in_command.go

```diff
@@ -35,8 +35,8 @@ func main() {
 	// though when having a out to in handover, it would be the version out had, maybe something else VersionFromFile
 	var version string = request.Version.Ref
 
-	var sourceUrl string = shared.InjectVersionIntoPath(request.Source.URI, version, "<version>")
-	URI, err := url.Parse(sourceUrl)
+	var SourceURL string = shared.InjectVersionIntoPath(request.Source.URI, version, "<version>")
+	URI, err := url.Parse(SourceURL)
 	if err != nil {
```

Comment:
uppercase?

Reply:
Was flagged by `golint` inline with convention(s) defined here: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Yes, URL should be upper case, but not Source, so we should have `sourceURL`

Reply:
You're right, thanks. Fixed.